### ISSUE TITLE
Image alignment in figure is not centered when image smaller than figcaption

### ISSUE THREAD

**Issue by nextguyover** [squidfunk/mkdocs-material#6731]
### Bug description

Creating a `figure` with an image that is smaller than the width of the `figcaption` prevents the image from being center aligned?

```
<figure markdown>
![](https://images.unsplash.com/photo-1619252584172-a83a949b6efd){ width="200" }
<figcaption>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</figcaption>
</figure>
```

![image](https://github.com/squidfunk/mkdocs-material/assets/56286401/d9eb658f-6265-45fb-96e4-d4940fa47965)

### Related links

- [Reporting a bug](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/)
- https://squidfunk.github.io/mkdocs-material/reference/images/#image-captions

### Reproduction

[figure-center-bug.zip](https://github.com/squidfunk/mkdocs-material/files/14133170/figure-center-bug.zip)

### Steps to reproduce

1. Unzip and run the project with `mkdocs serve`
2. Open page at `http://127.0.0.1:8000/`

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by alexvoss** [comment]
@squidfunk, does this text in the admonition in the [section on image aligment](https://squidfunk.github.io/mkdocs-material/reference/images/#image-alignment) mean that people should expect images in figures with captions are centered? 

> Instead, the [image captions](https://squidfunk.github.io/mkdocs-material/reference/images/#image-captions) syntax can be used, as captions are optional.

This is not default HTML behaviour, so should be more explicitly said that the CSS does this. The mechanism, I assume is the `text-align: center;` set for a figure?

It seems possible to use an empty caption to center an image that does not really need a caption. This seems to come out with a hight of zero but still has a space on the screen. Might that interfere with other things?

Once the issue is fixed, can we take the bit about centering images that have a caption out of the admonition? It seems too common a use case to hide it away. 

What I do to center images is to add [extra css](https://mkdocs-material.github.io/examples/page-status/stylesheets/extra.css). I assign images a class with [attribute lists](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#attribute-lists) and set the width of the image to something like 80% and then [define margins along these lines](https://www.w3schools.com/howto/howto_css_image_center.asp).

**Comment by squidfunk** [comment]
So, it's right that the content of the figure itself is currently not centered. I suppose that until now, users have only used images that are larger than the figure, which then gets shrunk via aspect ratio. I agree that the expected behavior for smaller images is certainly that the image is centered nonetheless, so we should change that in our CSS.

**Comment by squidfunk** [comment]
Fixed in 5b9bcb108

**Comment by squidfunk** [comment]
Released as part of [9.5.7](https://github.com/squidfunk/mkdocs-material/releases/tag/9.5.7).
